x-pack/plugins/ml/public/application/components/controls/select_severity/select_severity.test.tsx

```diff
@@ -37,7 +37,9 @@ describe('SelectSeverity', () => {
     );
 
     // assert initial state
-    expect(getAllByText('warning')).toHaveLength(2);
+    // EuiSuperSelect was refactored to update SR-only
```

Comment:
Suggestion to remove this comment in the code before merging

Reply:
@qn895 Sure thing. I'll remove this and merge. Thank you!

Reply:
I think I accidentally bumped your PR comment to a re-review. I've updated the test and CI is building again.